apps/explorer/priv/repo/migrations/20220617083341_set_counters_on_transaction_table.exs

```diff
@@ -0,0 +1,61 @@
+defmodule Explorer.Repo.Migrations.SetCountersOnTransactionTable do
+  use Ecto.Migration
+
+  def up do
+    execute("CREATE TABLE celo_transaction_stats(stat_type varchar(255), value bigint);")
```

Comment:
`bigint` looks too small for the total gas used value, no? I'd go with `numeric(100,0)` like in other similar cases.